src/lib/ld2410/types.ts

```diff
@@ -128,6 +140,16 @@ export type RangeGateSensitivityCommandPayload = {
   staticSensitivity: number;
 };
 
+export type BluetoothCommandPayload = {
+  type: "BLUETOOTH";
+  bluetooth: boolean;
```

Comment:
The following change makes the code a bit more self-explanatory that this is about the bluetooth power state and nothing else to do with bluetooth.

```suggestion
  enabled: boolean;
```